### PR TITLE
WIP: Adding Snippets

### PR DESCRIPTION
**Pull request by kcnakamu**

### CODE REVIEW

**Comment by kcnakamu** [comment]
https://mit-orcd.github.io/orcd-docs-previews/PR/PR57

File: docs/recipes/snippets-setup/mpi4py/OpenMind.sh
Comment (lmilechin) [inline docs/recipes/snippets-setup/mpi4py/OpenMind.sh:1]: If these have been added to the examples repo then we can delete these from this one.

**Review by lmilechin — CHANGES_REQUESTED** [review]
A few things to fix, once these are done we can probably merge!

**Review by lmilechin — APPROVED** [review]
Looks good!

**Pull request merged**